gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/util/ResourceUtil.java

```diff
@@ -0,0 +1,22 @@
+package uk.nhs.adaptors.pss.translator.util;
+
+import org.hl7.fhir.dstu3.model.Extension;
+import org.hl7.fhir.dstu3.model.IdType;
+import org.hl7.fhir.dstu3.model.Reference;
+import org.hl7.fhir.dstu3.model.Resource;
+
+public class ResourceUtil {
+
+    public static Reference buildResourceReference(Resource resource) {
```

Comment:
It seems this method is no longer needed, if we set Reference just from the resource (without using IdType).